apps/modernization-ui/src/pages/advancedSearch/components/patientSearch/AddressForm.tsx

```diff
@@ -2,74 +2,85 @@ import { Grid } from '@trussworks/react-uswds';
-                    control={control}
-                    name="zip"
-                    render={({ field: { onChange, value } }) => (
-                        <Input
-                            onChange={onChange}
-                            defaultValue={value}
-                            type="text"
-                            label="Zip code"
-                            htmlFor="zip"
-                            id="zip"
-                        />
-                    )}
-                />
-            </Grid>
+            <SearchCriteriaContext.Consumer>
+                {({ searchCriteria }) => (
+                    <>
+                        <Grid col={12}>
+                            <Controller
+                                control={control}
+                                name="address"
+                                render={({ field: { onChange, value } }) => (
+                                    <Input
+                                        onChange={onChange}
+                                        type="text"
+                                        label="Street address"
+                                        defaultValue={value}
+                                        htmlFor="address"
+                                        id="address"
+                                    />
+                                )}
+                            />
+                        </Grid>
+                        <Grid col={12}>
+                            <Controller
+                                control={control}
+                                name="city"
+                                render={({ field: { onChange, value } }) => (
+                                    <Input
+                                        onChange={onChange}
+                                        defaultValue={value}
+                                        type="text"
+                                        label="City"
+                                        htmlFor="city"
+                                        id="city"
+                                    />
+                                )}
+                            />
+                        </Grid>
+                        <Grid col={12}>
+                            <Controller
+                                control={control}
+                                name="state"
+                                render={({ field: { onChange, value } }) => (
+                                    <SelectInput
+                                        defaultValue={value}
+                                        onChange={onChange}
+                                        htmlFor={'state'}
+                                        label="State"
+                                        options={searchCriteria.states.map((state) => {
+                                            return {
+                                                value: state.codeDescTxt!,
```

Comment:
Based on what I'm seeing in the search API, you will need to use the value of `state.id`. However, this will display in the search as this: 
![image](https://user-images.githubusercontent.com/109251240/220756184-b883fb3b-030c-42dd-a924-931b187de465.png)

Not sure if this is a problem or if we need to change how those `Chip`s are displayed.